src/SCREENS.ts

```diff
@@ -24,4 +24,5 @@ export default {
     SIGN_IN_WITH_APPLE_DESKTOP: 'AppleSignInDesktop',
     SIGN_IN_WITH_GOOGLE_DESKTOP: 'GoogleSignInDesktop',
     DESKTOP_SIGN_IN_REDIRECT: 'DesktopSignInRedirect',
+    SAML_SIGN_IN: 'SAMLSignIN',
```

Comment:
NAB, Use lower case in value?
```suggestion
    SAML_SIGN_IN: 'SAMLSignIn',
```